priv/posts/performance-methodologies.md

```diff
@@ -0,0 +1,362 @@
+---
+title: Performance Methodologies
+date: 2021-01-21 10:00
+tags: performance,article,en-US
+---
+
+# Performance Methodologies
+
+Poor performance costs the software industry millions of dollars annually in lost revenue, decreased productivity, increased development and hardware costs, and damaged customer
+relations.
+
+Most applications tend to focus on correctness over performance. The shift towards performance only occurs once it becomes a problem.
+Once it becomes problem, one rarely has time to dedicate towards improving it. This article aims to show you that **there is no magic solution**.
+A lot of work should be done in early phases of development. This article will consider the reader having the role of  Performance Engineer or having to "act" as one.
+
+The reality is that some companies devotees of "agile" methodology, adopt the same order to every software development process: Idea -> MVP -> Feature -> "Refactor". One only needs some time in this methodology to understand that it's not quite like that.
+The result is that, somewhere near the end of the project, performance issues appear.
```

Comment:
```suggestion
In my experience, a strictly "agile" methodology (Idea -> MVP -> Feature -> "Refactor") tends to leave out proper performance engineering, since performance is not a goal but an expectation.
```